### DISCUSSION TITLE
可以只提取表格结构而不进行ocr吗？

### DISCUSSION THREAD

**Discussion by 102757017** [RapidAI/TableStructureRec#69] [General]
我只想提取表格的结构信息，为了加快速度不想运行ocr，table_engine(img_path)默认会执行ocr，严重拖累了识别速度，有办法禁用ocr吗？

**Comment by Joker1212** [comment]
打包的whl方案不行哈，可以自己从源代码改，下一版我加个参数吧，主要是只提取结构的话，输出的其实只有物理box和逻辑box，其实不具备实用性

**Reply by 102757017** [reply]
这个也是有应用场景的，我有许多多页文件，每份文件的首页封面都有相同的表结构。这些文件通过扫描仪扫描成了一个整体的pdf。为了将这个 PDF 文件分割成多个独立的文件，我们需要首先识别首页的表结构，以判断是否为封面。一旦确认是封面，就仅对封面进行 OCR 处理，而其他页面则无需进行 OCR。尽管对整个文件的每一页都进行 OCR 分析也能实现这一功能，但效率会大大降低。

**Reply by Joker1212** [reply]
嗯嗯，我改一下重新发包吧，加个执行时候的参数给有线和无线表格，控制一下是否进行ocr

**Comment by Joker1212** [comment]
代码已合并更新，下载新包生效
